apps/server/src/modules/h5p-editor/temporary-file-storage/temporary-file-storage.spec.ts

```diff
@@ -127,58 +130,80 @@ describe('TemporaryFileStorage', () => {
 	describe('deleteFile is called', () => {
 		describe('WHEN file exists', () => {
 			it('should delete file', async () => {
-				const { user, filename, filepath } = setup();
-				await service.deleteFile(filename, user.id);
-				const fileExists = existsSync(filepath);
-				expect(fileExists).toBe(false);
+				const { user1, file1 } = setup();
+				repo.findByPath.mockResolvedValue(file1);
+				await storage.deleteFile(file1.filename, user1.id);
+				expect(repo.delete).toHaveBeenCalled();
+				expect(s3clientAdapter.delete).toHaveBeenCalledWith([join(user1.id, file1.filename)]);
 			});
 		});
 		describe('WHEN file does not exist', () => {
 			it('should throw error', async () => {
-				const { user } = setup();
+				const { user1, file1 } = setup();
+				repo.findByPath.mockImplementation(() => {
+					throw new Error('Not found');
+				});
 				await expect(async () => {
-					await service.deleteFile('abc/nonexistingfile.txt', user.id);
+					await storage.deleteFile(file1.filename, user1.id);
 				}).rejects.toThrow();
+				expect(repo.delete).not.toHaveBeenCalled();
+				expect(s3clientAdapter.delete).not.toHaveBeenCalled();
 			});
 		});
 	});
 
 	describe('fileExists is called', () => {
 		describe('WHEN file exists', () => {
 			it('should return true', async () => {
-				const { user, filename } = setup();
-				await expect(service.fileExists(filename, user)).resolves.toBe(true);
+				const { user1, file1 } = setup();
+				repo.findByPath.mockResolvedValue(file1);
+				await expect(storage.fileExists(file1.filename, user1)).resolves.toBe(true);
```

Comment:
```suggestion
				const { user1, file1 } = setup();
				repo.findByPath.mockResolvedValue(file1);

				const result = await storage.fileExists(file1.filename, user1);

				expect(result).toBe(true);
```

It is nicer to have Act and Assert on two separate lines if possible.